### PR TITLE
[ADAM-1221] Bump Spark/Hadoop versions.

### PR DESCRIPTION
**Pull request by fnothaft**
Reverts #1254, then reapplies commit but while fixing sed replacement line.

### CODE REVIEW

**Comment by fnothaft** [comment]
This is still wrong!

File: scripts/move_to_spark_2.sh [hunk 1]
```diff
@@ -2,10 +2,12 @@
 
 set +x
 
-svp="\${scala\.version\.prefix}"
+svp="\${scala.version.prefix}"
 substitution_cmd="s/_$svp/-spark2_$svp/g"
 
 find . -name "pom.xml" -exec sed \
+    -e "/utils-/ s/_2\.10/-spark2_2.10/g" \
+    -e "/adam-/ s/_2\.10/-spark2_2.10/g" \
```
Comment (heuermh) [inline scripts/move_to_spark_2.sh]: Does this mean we're missing `${scala.version.prefix}` somewhere, or are we intentionally not using it somewhere, in artifactId perhaps?

Comment (fnothaft) [inline scripts/move_to_spark_2.sh]: That line of thought was what led me to delete it originally. However, yeah, we're intentionally not using SVP in the artifact names. That is what #1254 broke. This looks to be passing but I'd like to go over the CI logs before we merge.

**Comment by heuermh** [comment]
Thank you, @fnothaft!

**Pull request merged**